packages/cozy-stack-client/src/SharingCollection.js

```diff
@@ -28,28 +28,54 @@ class SharingCollection extends DocumentCollection {
    * @param  {string} sharingType - If "two-way", will set the open_sharing attribute to true
    * @param  {string} description - Describes the sharing
    * @param  {string=} previewPath Relative URL of the sharings preview page
+   * @param  {boolean} open_sharing If the sharing is open or not
```

Comment:
I can't understand from here or from the doc what an open sharing mean :/ I checked this doc: https://docs.cozy.io/en/cozy-stack/sharing/#post-sharings

Reply:
Not sure that the stack is the right doc to understand the details of the doctype. You can check here : https://github.com/cozy/cozy-doctypes/blob/master/docs/io.cozy.sharings.md

Reply:
I added a link from one doc to the other, I also found that searching with an underscore on the doc does not work correctly :/ https://github.com/cozy/cozy-stack/pull/2715